bin/sentry-wrap.py

```diff
@@ -69,30 +74,37 @@ def wrap_process(ctx, timeout, cmd):
 
     cmd = " ".join(cmd)
     cmd_args = shlex.split(cmd)
-    click.echo(f"Running: {cmd_args}")
+    if verbose:
+        click.echo(f"Running: {cmd_args}")
 
     try:
-        ret = subprocess.run(cmd_args, capture_output=True, timeout=timeout)
+        ret = subprocess.run(
+            cmd_args,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            timeout=timeout,
+        )
         if ret.returncode != 0:
             sentry_sdk.set_context(
                 "status",
                 {
                     "exit_code": ret.returncode,
-                    "stdout": ret.stdout.decode("utf-8"),
-                    "stderr": ret.stderr.decode("utf-8"),
+                    # Truncate stdout/stderr to the last 5,000 characters in case the
+                    # output is monstrously large.
+                    "stdout": ret.stdout.decode("utf-8")[-5000:],
```

Comment:
It's possible for the subprocess to generate a _lot_ of output. In practice, I don't think we use sentry-wrap in a way that this is likely to happen except for when we run migrations. It's possible a migration we wrote could have a lot of output.

It his truncates the output to the last 5000 characters before sending it to Sentry.

Reply:
I've never seen output truncated by number of character before -- only number of lines (like `--tail 10` returns the last 10 lines of a log file in `pm2`, a process manager I've used before).

Is there a possibility that a single line could be 5000 characters (or otherwise thousands of characters) long, such that this might not contain enough of the output for debugging?

Reply:
It is possible--everything is possible in a sense. I don't think we'll hit this issue with what we've got currently. But when/if we do, we'll see it in Sentry and we can figure out a better way of truncating. Maybe we truncate characters in a line and truncate lines, too.